tests/unit/test_simple_neuromodulation.py

```diff
@@ -0,0 +1,159 @@
+"""Test neuromodulations using the toy ringtest data as such:
+neuromodulatory projection: virtual_neurons->RingB, 0->0, 1->0
+The post synatic cell RingB gid 0 has 2 synapses:
+    RingA->RingB : 0->0
+    RingB->RingB : 1->0
+The neuromodulatory spikes are injected via replay from virtual neuron gid 1
```

Comment:
is it gid 1 or 1001? otherwise... how can you apply neuromodulation to the synapse ringA (0) to ringB (1000)?

Reply:
Here in the doc string are all SONATA gid (without offset), in the tests I checked with the offset.